templates/HowToStart.md

```diff
@@ -1,5 +1,5 @@
 ## Step-by-Step Guide
-1. Fork [this](https://github.com/keycryptovc/base) repository 
+1. Fork [this](https://github.com/keycryptovc/base) repository. In case you have forked [repository](https://github.com/keycryptovc/base) already, please update your fork before adding new projects. To keep repository up to date you could use [Upriver tool](https://upriver.github.io/)
```

Comment:
Perhaps it's better to split guides into two:

1. First PR
2. Followup PR's

?